src/helpers/provide/config.ts

```diff
@@ -1,7 +1,7 @@
 import { fetch, FetchMethods, FetchResultTypes } from '@sapphire/fetch';
 import { API_SECRET, API_URL, AUTOCODE_ENV } from './environment';
 import type { AutocodeAPIResponseModel } from '../../lib/model/AutocodeAPIResponseModel.model';
-import { container } from '@sapphire/framework';
+import { container } from '@sapphire/pieces';
```

Comment:
What is the advantage of `pieces` over `framework`?

Reply:
Importing pieces is much less resource-intensive because the module weighs much less than the framework.